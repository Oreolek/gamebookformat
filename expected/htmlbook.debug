BEGIN DEBUG OUTPUT
Book title: HTMLBook
Number of sections: 3
Introduction
 This is an introduction. This example gamebook is for testing the htmlbook option. See https://github.com/oreillymedia/HTMLBook for more information about HTMLBook. 
More
 This is another introduction section, just to see that formatting is correct when there are more than one. 

Turn to 1 to begin.
1 (third) -  Got to this section. Story ends here.
2 (second) -  Got to another section. Story ends here. 
3 (start) -  This is the first section. From here you can go to the unnamed section 2 or this named section (1). 
END DEBUG OUTPUT
