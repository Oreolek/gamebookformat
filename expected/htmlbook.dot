digraph gamebook {
3->2
3->1
}
