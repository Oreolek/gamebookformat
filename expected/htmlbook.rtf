{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf360
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
\paperw11900\paperh16840\margl1440\margr1440\vieww14140\viewh14860\viewkind0
\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\ql\qnatural\pardirnatural

\f0\b\fs24 \cf0

\b \qc HTMLBook
\b0\
\
\b \qc Introduction
\b0\



\ql  This is an introduction. This example gamebook is for testing the htmlbook option. See https://github.com/oreillymedia/HTMLBook for more information about HTMLBook.  \
\
\
\b \qc More
\b0\



\ql  This is another introduction section, just to see that formatting is correct when there are more than one.  \
\
\b Turn to 1 to begin.
\b0\
\
\b \qc 1
\b0\
\ql  Got to this section. Story ends here. \
\

\b \qc 2
\b0\
\ql  Got to another section. Story ends here.  \
\

\b \qc 3
\b0\
\ql  This is the first section. From here you can go to the unnamed section \b 2
\b0
 or this named section (
\b 1
\b0
)
.  \
\

}
