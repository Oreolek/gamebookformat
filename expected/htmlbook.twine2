<tw-storydata name="HTMLBook" startnode="1" creator="Twine" creator-version="2.0" format="Harlowe" options="">
<style role="stylesheet" id="twine-user-stylesheet" type="text/twine-css"></style><script role="script" id="twine-user-script" type="text/twine-javascript"></script>
<tw-passagedata pid="1" name="1" tags="" position="10,10"> Got to this section. Story ends here.</tw-passagedata>
<tw-passagedata pid="2" name="2" tags="" position="20,20"> Got to another section. Story ends here. </tw-passagedata>
<tw-passagedata pid="3" name="3" tags="" position="30,30"> This is the first section. From here you can go to the unnamed section [[2]]
 or [[this named section->1]]
. </tw-passagedata>
</tw-storydata>
