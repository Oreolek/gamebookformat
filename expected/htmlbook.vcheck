examples/htmlbook.debug: ending (death?) section found: second
examples/htmlbook.debug: ending (death?) section found: third
