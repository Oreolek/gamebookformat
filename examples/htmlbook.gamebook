title = HTMLBook
= Introduction
This is an introduction.
This example gamebook is for testing the htmlbook option.
See https://github.com/oreillymedia/HTMLBook for more information about HTMLBook.

= More
This is another introduction section, just to see that formatting is correct when
there are more than one.

* start
This is the first section. From here you can go to the unnamed section [[second]]
or [[third][this named section]].

* second
Got to another section. Story ends here.

* third
Got to this section. Story ends here.
