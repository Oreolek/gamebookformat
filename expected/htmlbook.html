<!DOCTYPE html>

<html xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 xsi:schemaLocation="http://www.w3.org/1999/xhtml ../../HTMLBook/schema/htmlbook.xsd"
 xmlns="http://www.w3.org/1999/xhtml">

<head>
 <title>HTMLBook</title>
 <meta name="HTMLBook" content="text/html; charset=utf-8" />
</head>
<body data-type="book" class="book" id="htmlbook">
 <section data-type="titlepage" class="titlepage">
    <h1>HTMLBook</h1>
    <p data-type="author"></p>
 </section>
<section data-type="chapter" id="sections">
<h1>HTMLBook</h1>
<section data-type="sect1" class="sect1">
<h1>Introduction</h1>
<p> This is an introduction. This example gamebook is for testing the htmlbook option. See https://github.com/oreillymedia/HTMLBook for more information about HTMLBook. </p>
</section>

<section data-type="sect1" class="sect1">
<h1>More</h1>
<p> This is another introduction section, just to see that formatting is correct when there are more than one. </p>
</section>


<section data-type="sect1" class="sect1" id="section1">
<h1>1</h1>
<p> Got to this section. Story ends here.</p>
</section>
<section data-type="sect1" class="sect1" id="section2">
<h1>2</h1>
<p> Got to another section. Story ends here. </p>
</section>
<section data-type="sect1" class="sect1" id="section3">
<h1>3</h1>
<p> This is the first section. From here you can go to the unnamed section <a href="#section2">2</a>
 or <a href="#section1">this named section</a>
. </p>
</section>
    </section>
  </body>
</html>
